### PR TITLE
Jd 2022 10 react fixes

### PR DESCRIPTION
**Pull request by goapunk**
* remove ref from component initialization as it doesn't work for functional components
* rename index.jsx of comments to react_comments.jsx to be more in line with the other components
* move polls from assets/ to static/ to be in line with other components

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ import './adhocracy4/static/global_jquery'
 export { default as alert } from './adhocracy4/static/Alert'
 export { default as api } from './adhocracy4/static/api'
 export { default as comments } from './adhocracy4/comments/static/comments/react_comments'
-export * as commentsAsync from './adhocracy4/comments_async/static/comments_async'
+export * as commentsAsync from './adhocracy4/comments_async/static/comments_async/react_comments'
```
Comment (phillimorland) [inline index.js:9]: so we don't double up names I would call this react_comments_async or something?

**Review by phillimorland — COMMENTED** [review]
looks good but one nameing thing and may cause conflicts with poll Pr which is ready again

File: index.js (hunk 1 above)
Comment (goapunk) [reply index.js]: done

File: index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export * as commentsAsync from './adhocracy4/comments_async/static/comments_asyn
 export { default as config } from './adhocracy4/static/config'
 export { default as follows } from './adhocracy4/follows/static/follows/react_follows'
 export * as maps from './adhocracy4/maps/static/a4maps/a4maps_common'
-export { default as polls } from './adhocracy4/polls/assets/react_polls'
```
Comment (phillimorland) [inline index.js:13]: @goapunk sorry missed this before, need to also rename the folder :p then feel free to merge

Comment (goapunk) [reply index.js:13]: sorry, don't understand?

Comment (goapunk) [reply index.js:13]: ahh, forgot to commit :man_facepalming:

**Review by phillimorland — APPROVED** [review]
Nice

**Pull request merged**